src/aliases.jl

```diff
@@ -48,3 +50,7 @@ function number_of_weak_compositions end
 @alias n_generators ngens
 @alias n_rows nrows
 @alias n_variables nvars
+
+# for backwards compatibility
+@alias dualgraph dual_graph
+@alias edgegraph vertex_edge_graph
```

Comment:
Shouldn't these rather be `@deprecate`? Anyway, if they should be phased out they should be in `deprecations.jl` and not `aliases.jl`. `aliases.jl` only contains the aliases that we plan to keep forever

Reply:
We discusses this internally and this is the result.